physical_infra_provider_configuration/configuration/topics/index.adoc

```diff
@@ -1 +1,6 @@
-== Configuring the Physical Infrastructure Provider
\ No newline at end of file
+== Configuring Lenovo XClarity Administrator
+
+Lenovo XClarity Administrator virtual appliances are connected to CloudForms by adding physical-infrastructure providers. Physical-infrastructure providers enable you to manage and interact with devices that are managed by Lenovo XClarity Administrator. After the physical-infrastructure provider is added, the servers that are managed by the Lenovo XClarity Administrator are registered in CloudForms. Managed devices can be monitored, configured, and updated using Lenovo XClarity Administrator, and the physical-infrastructure provider can surface details for the associated Lenovo XClarity Administrator  instance using the CloudForms user interface.
```

Comment:
The last sentence here is wordy. Can we simplify this?     Managed devices can be monitored, configured and updated in Lenovo XClarity Administrator.  The CloudForms provider surfaces information about these managed devices and the XClarity Administrator instance.